project/JavalibIRCleaner.scala

```diff
@@ -204,7 +204,7 @@ final class JavalibIRCleaner(baseDirectoryURI: URI) {
         case m @ MethodDef(flags, name, originalName, args, resultType, body) =>
           implicit val pos = m.pos
           MethodDef(flags, transformMethodIdent(name), originalName, transformParamDefs(args),
-              transformType(resultType), body)(m.optimizerHints, m.hash)
+              transformType(resultType), body)(m.optimizerHints, Version.NoVersion)
```

Comment:
Note: We're running the overall result through Hashers anyways (as we should). So this will just keep an old / wrong hash (in fact, it could lead to spurious diffs where there are non in the resulting tree).